Applications/Games/Lego Rock Raiders/Local/script.js

```diff
@@ -0,0 +1,56 @@
+include(["engines", "wine", "quick_script", "local_installer_script"]);
+include(["utils", "functions", "net", "download"]);
+include(["utils", "functions", "net", "resource"]);
+include(["utils", "functions", "filesystem", "extract"]);
+include(["utils", "functions", "filesystem", "files"]);
+include(["engines", "wine", "plugins", "regsvr32"]);
+include(["engines", "wine", "verbs", "quartz"]);
+
+var installerImplementation = {
+    run: function() {
+        new LocalInstallerScript()
+            .name("Lego Rock Raiders")
+            .editor("LEGO Media")
+            .author("Zemogiter")
+            .category("Games")
+            .executable("LegoRR.exe")
+            .preInstall(function(wine) {
+                wine.quartz();
+            })
+            .postInstall(function(wine,wizard) {
+                var GameDir = wine.prefixDirectory() + "drive_c/" + wine.programFiles() + "/LEGO Media/Games/Rock Raiders/d3drm.dll";
```

Comment:
variable names should be camel case: `gameDir`

Apart from this, the variable does not contain the game dir but the dll path, so please use a matching name.

Reply:
But the script works no matter if variable names start with capital letter or not.